metrics/metrics_test.go

```diff
@@ -13,7 +12,7 @@ const FANOUT = 128
 
 // Stop the compiler from complaining during debugging.
 var (
-	_ = ioutil.Discard
+	//_ = ioutil.Discard
```

Comment:
please just remove this line

Reply:
<img width="449" alt="image" src="https://github.com/dominant-strategies/go-quai/assets/47848969/9ea0de62-0568-45f2-89e7-27f8f359316f">

> please just remove this line

Yeah sure, I'll change and remove and roll back the changes and merge my commits now